[Python] Concepts naming cleanup and typo patrol (#2936)
exercism/v3: languages/python/concepts/lists/about.md

'Oatmeal'
```

> You can also access a portion of a list with _slice notation_ (`[start:stop]`). A _slice_ is defined as the sequence of items in a list at position `index` such that `start <= index < stop`. _Slicing_ does not modify the original `list`. Instead, you get a new list with copies of all the elements you asked for.

Reviewer: ```suggestion
You can also access a portion of a list with _slice notation_ (`[start:stop]`). A _slice_ is defined as the sequence of items in a list at position `index` such that `start <= index < stop`. _Slicing_ does not modify the original `list`. Instead, you get a new list with (only) the elements you asked for.
```
